infra/tools/pr_helper/action.yml

```diff
@@ -0,0 +1,23 @@
+name: 'PR helper'
+description: 'Add comments to indicate if authors are authorized.'
+inputs:
+  githubToken: 
+    description: 'The github action token.'
+    required: true
+    default: ''
```

Comment:
Maybe don't include a default for required fields so that there's an error thrown (i assume) if it's not provided.